app/code/AndriiShkrebtii/RegularCustomer/view/frontend/web/js/message.js

```diff
@@ -0,0 +1,25 @@
+define([
+    'jquery',
+    'jquery/ui'
+], function ($) {
+    'use strict';
+
+    $.widget('andriiShkrebtii.regularCustomerMessage', {
+        /**
+         * Constructor
+         * @private
+         */
+        _create: function () {
+            $(document).on('andrii_shkrebrii_regular_customer_show_message', this.showMessage.bind(this));
+        },
+
+        /**
+         * Generate event to show message
+         */
+        showMessage: function () {
+            $(this.element).css('display', 'inline-block');
```

Comment:
@jimmywilson111 Do not decide in JS  which type of display to use. This is for CSS. Should add `display:none;` or remove it, but CSS rules must be written in the LESS files/. In JS you should better operate visibility and nothing else.

Reply:
Done ✔